membership_delegated_partner/models/membership_line.py

```diff
@@ -1,12 +1,26 @@
 # Copyright 2017 Tecnativa - David Vidal
+# Copyright 2019 Onestein - Andrea Stirpe
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, models
+from odoo import api, models, fields
 
 
 class MembershipLine(models.Model):
     _inherit = 'membership.membership_line'
 
+    partner = fields.Many2one(
+        compute='_compute_partner', store=True, readonly=False)
```

Comment:
Maybe this aproach is too 13ish for this version :sweat_smile:

Reply:
Indeed in V13 doesn't break the tests :laughing: 
I'll take a look at it tomorrow...